wimm.Guardian.UnitTests/PropertyTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace wimm.Guardian.UnitTests
+{
+    [TestClass]
+    public class PropertyTests
+    {
+        [TestMethod]
+        public void Property_ExpressionAccessesProperty_ReturnsPropertyName()
+        {
+            var dummy = new Dummy(123);
+            var underTest = dummy.Require(nameof(dummy)).Property(x => x.Doot);
+            Assert.AreEqual("dummy.Doot", underTest.Name);
+        }
+
+        [TestMethod]
+        public void Property_ExpressionAccessProperty_ReturnsPropertyValue()
+        {
+            var dummy = new Dummy(123);
```

Comment:
might prefer member initialization here. not that it's a big deal but I had to scroll to read the ctor to be sure the test made sense.

Reply:
I can see the merit for it & if it wasn't a 40 line file I'd be likely to do it. I'd rather just get this guy in for now though. If the class gets bigger I'll make that change

Reply:
sounds good